README.md

```diff
@@ -1 +1,16 @@
-# Basic Calculator
+# [itp-w1] Highest Number Cubed
+
+Sometimes you can use programming to solve problems that would otherwise be difficult.
+For example, can you think of the highest number that, when cubed, is less than 12,000?
+Even using a calculator you would probably have to manually guess numbers until you figure it out. 
+Write a program that receives an upper limit and figures this out for you!
+
+All students will fork this repo and set up their cloud9 for this project individually with guidance from the mentor.
```

Comment:
I think the plan was to have the mentor walk them through it. Do you have a file already written with the instructions somewhere that I could copy paste it to here?